views/pages/registry.hbs

```diff
@@ -15,5 +15,12 @@
                 <img src="/img/amazon.png">
             </a>
         </div>
-    </div>
+    </div>    
+    
+    <p>
+        We are registered at <a href="https://www.crateandbarrel.com/gift-registry/zainab-zahra-and-ahmed-jafri/r5842602">Crate and Barrel</a>
+        <p>
+        We are registered at <a href="https://www.myregistry.com/wedding-registry/Zainab-Zahra-and-Ahmed-Jafri-REDMOND-WA/1597552">World market</a>
```

Comment:
Please use capitalization for both `World` and `Market`.

Reply:
Acha fine